.circleci/config.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           command: |
             set +e
             TESTFILES=$(circleci tests glob "src/**/*.test.ts*" | circleci tests split --split-by=timings)
-            npx yarn test:circleci $TESTFILES
+            npx yarn test:circleci $TESTFILES --forceExit --detectOpenHandles
       - store_test_results:
```

Comment:
What's the thinking behind this change?

Per jest docs, there's a performance penalty associated with enabling this:
https://jestjs.io/docs/cli#--detectopenhandles